.github/workflows/publish.js.yml

```diff
@@ -65,10 +80,76 @@ jobs:
         popd
         mv $PKG_PATH_TVOS/Build/Products/Debug-appletvos/$ZIP_PKG_NAME_TVOS ./
 
-    # release tasks
+    # building WDA packages for x86_64
+    - name: Build iOS for simulator for x86_64
```

Comment:
I would say we can still properly design that by having the loop in the script itself
E.g. design it as a single step and then iterate through platforms using vanilla bash for loop. This way we could reduce unnecessary duplication a lot

It is also possible to have a separate shell script for that and just run it from here

Reply:
Yea, am thinking to modify existing Scripts/build.sh to do the same script after this to simplify more